docker/.env

```diff
@@ -6,4 +6,5 @@ CLIENT_SECRET='xx'
 OAUTH_SECRET='random-secret-string'
```

Comment:
This file can be auto-generated and managed by `dtaas config` subcommand. The source of truth is the `dtaas.toml`.